src/models/exclude_filter.rs

```diff
@@ -25,7 +25,7 @@ pub(crate) struct Constraint {
   queries: Vec<String>,
```

Comment:
```suggestion
  exclude: Vec<String>,
```
Let's rename `queries` to `exclude`, what do you think? 

I was thinking later when we want to implement #309, we can add a field called `includes` and implement its semantics.

Reply:
We could do that, or we can also address this once we start implementing `includes`?